integration/app/app.component.html

```diff
@@ -0,0 +1,2 @@
+<app-interval *ngIf="shown"></app-interval>
```

Comment:
We should try it with multiple instances and see that one doesn't affect the others.